templates/subscription.template.json

```diff
@@ -171,6 +171,15 @@
         "environmentVariable": "apimEndpointSubnetCount"
       }
     },
+    "sharedWorkerSubnetCount": {
+      "type": "int",
+      "defaultValue": 0,
```

Comment:
Why is the default value different to environment.template.json?

Reply:
Set to 1 along with other subnet count parameters